### PR TITLE
Release 4.7.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.7.0](https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.7.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.7.0 (2021-09-08)
+
+- Added ability to place an element at the bottom of another element or before another element when using the dock service and the dynamic component service. [#250](https://github.com/blackbaud/skyux-core/pull/250)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Added the ability to place an element at the bottom of another element or before another element when using the dock service and the dynamic component service. [#250](https://github.com/blackbaud/skyux-core/pull/250)
```

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Also, `place an element at the bottom of another element or before another element` seems oddly worded to me, and I can't tell what we mean by "before." Could we change to `Added the place elements below or above other elements when...`? Or does that change the meaning?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.7.0](https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.7.0)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**